Cargo.toml

```diff
@@ -14,21 +14,23 @@ exclude = ["/.github", "/examples", "/scripts"]
 
 [dependencies]
 tokio = "1.0"
-rustls = { version = "=0.22.0-alpha.2", default-features = false }
+rustls = { version = "0.22.0-alpha.2", default-features = false }
 
 [features]
 default = ["logging", "tls12"]
-dangerous_configuration = ["rustls/dangerous_configuration"]
 early-data = []
 logging = ["rustls/logging"]
 secret_extraction = ["rustls/secret_extraction"]
 tls12 = ["rustls/tls12"]
 
 [dev-dependencies]
-argh = "0.1"
+argh = "0.1.1"
 tokio = { version = "1.0", features = ["full"] }
 futures-util = "0.3.1"
-lazy_static = "1"
+lazy_static = "1.1"
 webpki-roots = "=0.26.0-alpha.1"
 rustls-pemfile = "=2.0.0-alpha.1"
-webpki = { package = "rustls-webpki", version = "=0.102.0-alpha.2", features = ["alloc", "std"] }
+webpki = { package = "rustls-webpki", version = "=0.102.0-alpha.3", features = ["alloc", "std"] }
```

Comment:
Please submit one specific commit (or if you have don't have experience wrangling commits, one PR), per logical change.

Reply:
Reason ditto for the rustls patch change because of this (https://github.com/rustls/rustls/blob/86aa146d59f83e9fbf890b05f823799c0e8d3023/rustls/Cargo.toml#L22C5-L22C5). The intent is to track the latest main branch, and not really going after the crates.io version.